device/core/src/client.ts

```diff
@@ -493,7 +235,15 @@ export class Client extends EventEmitter {
    */
   sendEventBatch(messages: Message[], sendEventBatchCallback?: (err?: Error, result?: results.MessageEnqueued) => void): void {
     /*Codes_SRS_NODE_DEVICE_CLIENT_05_008: [The sendEventBatch method shall send the list of events (indicated by the messages argument) via the transport associated with the Client instance.]*/
-    this._fsm.handle('sendEventBatch', messages, sendEventBatchCallback);
+    if (this._transport.sendEventBatch) {
```

Comment:
>(this._transport.sendEventBatch) [](start = 7, length = 32)

Two different ways to determine if a capability is present in the  transport?

Reply:
yeah. I hate that. I'll fix it in a separate PR though.

---
In reply to: [145258076](https://github.com/Azure/azure-iot-sdk-node/pull/109#discussion_r145258076) [](ancestors = 145258076)